nss-day-cohort-29/nutshell-midnight-dream
# Conditional "Delete" button and article styling

# Description
"Delete" button is now conditional and only shows for the logged in user. Also applied styling (cornsilk bg and italicized title) to friends' articles.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout news-delete-button-conditional`
3. Run grunt
4. Update database with data below
4. Log in using `userName2` and `userEmail2@gmail.com`
5. Verify that "Delete" button only shows on user2's articles (userName says "You")
6. Verify that friends' articles are styled w/ cornsilk bg and italicized title.

```
{
  "users": [
    {
      "id": 1,
      "name": "userName",
      "email": "userEmail@gmail.com"
    },
    {
      "id": 2,
      "name": "userName2",
      "email": "userEmail2@gmail.com"
    },
    {
      "name": "userName3",
      "email": "userEmail3@gmail.com",
      "id": 3
    },
    {
      "name": "userName4",
      "email": "userEmail4@gmail.com",
      "id": 4
    }
  ],
  "events": [
    {
      "id": 1,
      "name": "eventName",
      "date": "eventDate",
      "location": "eventLocation",
      "userId": 1
    },
    {
      "name": "New Events",
      "date": "New Date",
      "location": "New Location",
      "userId": 2,
      "id": 2
    },
    {
      "name": "Watch a play",
      "date": "2019-01-12",
      "location": "Home",
      "userId": 2,
      "id": 3
    }
  ],
  "articles": [
    {
      "id": 1,
      "title": "User 1 Article (fourth)",
      "url": "http://google.com",
      "synopsis": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus efficitur purus eget est rutrum tincidunt. Vestibulum a velit accumsan quam scelerisque efficitur. Proin tincidunt magna id orci ultricies vulputate. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Duis laoreet lacus nibh, id tincidunt enim euismod in. Sed egestas sapien orci, in vestibulum ipsum tempor id. Aenean eleifend mi dui, in consequat lacus laoreet quis.",
      "timestamp": "1457395221082",
      "userId": 1
    },
    {
      "id": 2,
      "title": "User 2 Article (second)",
      "url": "http://google.com",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "timestamp": "1247395221082",
      "userId": 2
    },
    {
      "id": 3,
      "title": "User 2 Article2 (third)",
      "url": "http://google.com",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "timestamp": "1447395221082",
      "userId": 2
    },
    {
      "id": 4,
      "title": "User 3 Article (first)",
      "url": "http://google.com",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "timestamp": "1047395221082",
      "userId": 3
    },
    {
      "id": 5,
      "title": "User 4 Article",
      "url": "http://google.com",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "timestamp": "1647495221082",
      "userId": 4
    },
    {
      "title": "timestamp test (fifth)",
      "synopsis": "time!",
      "url": "meg.com",
      "timestamp": 1547395221082,
      "date": "01/13/2019 at 10:0",
      "userId": 2,
      "id": 6
    },
    {
      "title": "another timestamp (sixth)",
      "synopsis": "time time time",
      "url": "meg.com",
      "timestamp": 1547395320738,
      "date": "01/13/2019 at 10:2",
      "userId": 2,
      "id": 7
    },
    {
      "title": "it's a new hour",
      "synopsis": "woo!",
      "url": "meg.com",
      "timestamp": 1547402892179,
      "date": "01/13/2019 at 12:08",
      "userId": 2,
      "id": 10
    },
    {
      "title": "user2 @ 11:46",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "url": "meg.com",
      "timestamp": 1547488036316,
      "date": "1/14/2019 at 11:47AM",
      "userId": 2,
      "id": 11
    },
    {
      "title": "user1 @ 11:47",
      "synopsis": "Fusce turpis mi, sollicitudin quis maximus ac, facilisis vel lectus. Duis laoreet sollicitudin odio at viverra. Phasellus scelerisque, ante id lacinia cursus, elit nunc ullamcorper ipsum, vel faucibus sapien lectus nec nisl. In dui lorem, dignissim sit amet turpis pulvinar, congue malesuada ligula. Morbi dignissim, est vel lacinia mollis, magna ligula posuere arcu, rhoncus ultricies est dui vel risus.",
      "url": "meg.com",
      "timestamp": 1547488061390,
      "date": "1/14/2019 at 11:47AM",
      "userId": 1,
      "id": 12
    }
  ],
  "tasks": [],
  "messages": [
    {
      "id": 1,
      "message": "messageContent",
      "time": "messageTime",
      "userId": 1
    }
  ],
  "connections": [
    {
      "id": 1,
      "userId": 1,
      "otherId": 2
    },
    {
      "id": 2,
      "userId": 1,
      "otherId": 3
    },
    {
      "id": 3,
      "userId": 2,
      "otherId": 1
    },
    {
      "id": 4,
      "userId": 2,
      "otherId": 3
    }
  ]
}
```

# Checklist:
